src/Grid/Grid.php

```diff
@@ -1764,6 +1788,11 @@ public function setDefaultTweak($tweakId)
         return $this;
     }
 
+    public function getDefaultTweak(): ?string
```

Comment:
Similar to above, does string make the most sense here?

Reply:
Yes